tools/scripts/gen-env-files.sh

```diff
@@ -434,11 +434,11 @@ if test -f "./services/syndication/.env"; then
     echo "./services/syndication/.env exists"
 else
 echo \
-"KEYCLOAK_AUTH_SERVER_URL=http://host.docker.internal:$portKeycloak/auth/
-
-DB_URL=jdbc:postgresql://host.docker.internal:$portDatabase/$dbName
-DB_USERNAME=$dbUser
-DB_PASSWORD=$password
+"API_HOST_URL=http://host.docker.internal:$portApi
```

Comment:
The other services will need to be updated to use the API, but I have little interest in doing this now that I'm going to refactor them into .NET.